packages/react-components/react-data-grid-react-window/package.json

```diff
@@ -32,6 +32,8 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@fluentui/react-table": "9.0.0-alpha.19",
```

Comment:
updates dependencies to follot @Hotell 's recommendation https://github.com/microsoft/fluentui/pull/26370#issuecomment-1385177764